Src/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -35,11 +35,15 @@ public class GoogleAuthorizationCodeFlow : AuthorizationCodeFlow
         /// <summary>Gets the token revocation URL.</summary>
         public string RevokeTokenUrl { get { return revokeTokenUrl; } }
 
+        /// <summary>Gets or sets the include granted scopes indicator.</summary>
+        private bool? includeGrantedScopes { get; set; }

```

Comment:
I would prefer us to expose a public getter, similar to the way revokeTokenUrl is declared:

private readonly bool includeGrantedScopes;

/// <summary>Gets or sets the include granted scopes indicator.</summary>
private bool? IncludeGrantedScopes { get { return includeGrantedScopes; } }